packages/internals/package.json

```
@@ -90,5 +90,13 @@
     "arg": "5.0.2",
     "prompts": "2.4.2"
   },
+  "peerDependencies": {
+    "typescript": ">5.1.0"
```

According to the docs

```suggestion
    "typescript": ">=5.1.0"
```